tests/components/roborock/test_number.py

```diff
@@ -0,0 +1,57 @@
+"""Test Roborock Number platform."""
+from unittest.mock import patch
+
+import pytest
+from roborock.exceptions import RoborockException
+
+from homeassistant.components.number import ATTR_VALUE, SERVICE_SET_VALUE
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.parametrize(
+    ("entity_id", "value"),
+    [
+        ("number.roborock_s7_maxv_volume", 3.0),
```

Comment:
Can we also test minimum, maximum and some value in between here?

Reply:
The api itself is patched, so passing 3.0 vs any other number will give the same results. It's just checking that is called correctly as a paramater.